### DISCUSSION TITLE
Copilot Error 403

### DISCUSSION THREAD

**Discussion by lbaiao** [community/community#49787] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I'm getting the following error on Copilot for VS Code. I have a subscription. How do I solve this?

[INFO] [auth] [2023-03-11T11:43:37.757Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-03-11T11:43:37.759Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."

**Comment by iamcjwilliams** [comment, marked as answer]
@lbaiao Have you visited here to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Comment by lbaiao** [comment]
It worked. Thanks!

Em seg., 13 de mar. de 2023 às 13:35, iamcjwilliams <
***@***.***> escreveu:
